Makefile

```diff
@@ -1,13 +1,25 @@
 .PHONY: build clean lint deploy-production
 
 build:
-	cd electronic-tariff-file-rotations && env GOARCH=amd64 GOOS=linux CGO_ENABLED=0 go build -ldflags="-s -w" -o ../bin/handler
+	cd electronic-tariff-file-rotations && env GOARCH=amd64 GOOS=linux CGO_ENABLED=0 go build -ldflags="-s -w" -o ../bootstrap
```

Comment:
go compiles to a binary that can run anywhere so there's zero need for a dedicated runtime